README.md

```diff
@@ -29,9 +29,12 @@ This method allows us to thoroughly test queries within a sharded environment, e
 
 ## How to use
 
-Build the `vitess-tester` binary:
+After installing the `vitess-tester` binary, you need to have Vitess installed and in your path. 
```

Comment:
Reading this and looking at the new `./build.env` file, I understand that we expect people to add the path to the Vitess binaries in their `$PATH`. In which case should probably explain why, since simply building the vitess binaries and doing a `source build.env` in the vitess repository won't work. Or at least link them to this guide: https://vitess.io/docs/20.0/get-started/local/#install-vitess.